src/docs/vercel/index.md

```diff
@@ -0,0 +1,63 @@
+# Setting up Vercel with is-a.dev subdomain
+
+This guide will walk you through the process of setting up a Vercel site and pointing your is-a.dev subdomain to it.
+
+## Make sure you have a Vercel site.
+If you haven't made a Vercel site, make sure to make one. Follow the instructions in [Vercel's Docs](https://vercel.com/docs/getting-started-with-vercel).
+
+## Creating the domain files.
+First, go to your Vercel dashboard, then go to your project, then go to Custom Domains and put in there the is-a.dev subdomain you want.
+
+**You will need to make two files to ensure this proccess goes smoothly**. We will go in order for this section.
+
+1. Make a file for Vercel TXT verification.
+When connecting the domain, you will be greeted with a TXT verification. To make the file for this one it should be `_vercel.subdomain.json` in the **domains directory** (replace subdomain with the domain you want of course) and the file should be like this (You can leave email field blank as long as you have another social, but **DON'T REMOVE THE FIELD**.):
+```json
+{
+  "owner": {
+    "username": "insertusername",
+    "email": ""
+    "discord": "insertdiscordid"
+  },
+  "record": {
+    "TXT": "inserttxtstring"
+  }
+}
+```
+**DON'T MAKE A PR YET**, we still have to make another file. If you were to make a PR until this point, we will reject your domain since you are trying to make a nested subdomain on a subdomain you don't own yet. Please go to the next step.
```

Comment:
```suggestion
***DON'T MAKE A PULL REQUEST YET***, we still have to make another file. If you were to make a pull request at this point, we would reject your domain since you are trying to make a nested subdomain on a subdomain you don't own yet. Please proceed to the next step.
```